subprojects/platform-base/src/main/java/org/gradle/platform/base/plugins/BinaryBasePlugin.java

```diff
@@ -68,13 +69,14 @@ void registerBaseBinarySpec(TypeBuilder<BinarySpec> builder) {
         @Mutate
         void copyBinaryTasksToTaskContainer(TaskContainer tasks, BinaryContainer binaries) {
```

Comment:
does the injection work if we use `TaskContainerInternal` here?

Reply:
Nope.